### ISSUE TITLE
Unable to bypass Cortex XDR

### ISSUE THREAD

**Issue by KelvinSiu-HK** [0xsp-SRD/mortar#11]
bin.enc is an encrypted CS Beacon, tried to create the following batch file and launch it. 

@echo off 
cmd.exe /c rundll32.exe agressor.dll,stealth

Beacon connection was failed and Cortex XDR blocked with "Rule ioc.cobalt_strike_named_pipe"
![issue](https://user-images.githubusercontent.com/9109177/150643284-14bbb970-1653-4d67-8a38-f90e8a3d4bfc.PNG)

Any idea? Thanks.

**Comment by lawrenceamer** [comment]
Cortex respond to this with a fix, currently, i am working on a new bypass to push into my early access repo

**Comment by Cy-Beard** [comment]
I see the repo has been updated in May. Was a fix also added?
I also attempted this in a live environment for purple teaming engagements and was unsuccessful. But I am unsure if its the rule itself or cortex itself that flagged it.
![image](https://user-images.githubusercontent.com/41340083/187935155-8f983a10-2a4a-4572-b027-a64597cd7316.png)
Metasploit was able to create the session but once cortex flagged it, it killed it as well.
![image](https://user-images.githubusercontent.com/41340083/187935366-67270448-4bcc-4df4-bf7d-c0e8de5a3946.png)
